src/components/UI/Icon/Icon.tsx

```diff
@@ -207,39 +207,39 @@ export const IconAddFile: React.FunctionComponent<SVGProps<SVGElement>> = ({ cla
     >
       <path
         d="M10.5712 19.2247C13.032 19.2247 14.2417 19.2247 16.7025 19.2247C16.7951 19.2211 17.255 19.2065 17.6188 18.8056C17.8522 18.5468 17.9998 18.1896 17.9998 17.7923V5.97038L13.7201 1H6.09075C5.3769 1 4.79346 1.64151 4.79346 2.42882C4.79346 5.01673 4.79346 4.62506 4.79346 7.21297"
-        stroke="#3B8CC5"
+        stroke="#2D5FAA"
         strokeWidth="1.9615"
```

Comment:
use `stroke` [classes](https://tailwindcss.com/docs/stroke)?

Reply:
but its in the svg path tag.

Reply:
suggestion (non-blocking): u can group `path` with `g` and apply class to it